### ISSUE TITLE
Incomptability with Paradise Lost.

### ISSUE THREAD

**Issue by Baconbacon123** [TerraformersMC/Terrestria#289]
Fails to launch when used with https://github.com/devs-immortal/Paradise-Lost. This is the error message:
![image](https://user-images.githubusercontent.com/87005234/205423949-14bf5534-76ac-4d15-bcaa-c7a832d16005.png)

**Comment by gniftygnome** [comment]
Paradise Lost uses MoreTags and MoreTags does this:

```java
@Mixin(CropBlock.class)
public abstract class CropBlockMixin {
    @Redirect(
        method = "getAvailableMoisture",
        at = @At(
            value = "INVOKE",
            target = "Lnet/minecraft/block/BlockState;isOf(Lnet/minecraft/block/Block;)Z"
        ),
        slice = @Slice(
            from = @At("HEAD"),
            to = @At(
                value = "INVOKE",
                target = "Lnet/minecraft/block/BlockState;get(Lnet/minecraft/state/property/Property;)Ljava/lang/Comparable;"
            )
        )
    )
    private static boolean isBlockFarmland(BlockState blockState, Block block) {
        return blockState.isIn(MoreBlockTags.FARMLAND);
    }
```

This is probably actually a conflict with Terraform API.

**Comment by gniftygnome** [comment]
Yes, definitely.  Terraform API does this:

```java
@Mixin(CropBlock.class)
public class MixinCropBlock {
	@Redirect(method = "Lnet/minecraft/block/CropBlock;getAvailableMoisture(Lnet/minecraft/block/Block;Lnet/minecraft/world/BlockView;Lnet/minecraft/util/math/BlockPos;)F", at = @At(value = "INVOKE", target = "Lnet/minecraft/block/BlockState;isOf(Lnet/minecraft/block/Block;)Z"))
	private static boolean onGetAvailableMoisture(BlockState state, Block block) {
		return state.isOf(block) || (block == Blocks.FARMLAND && state.isIn(TerraformDirtBlockTags.FARMLAND));
	}
}
```

**Comment by gniftygnome** [comment]
Resolved in our 1.19.4 versions, in the sense it will no longer crash.  If Paradise Lost's mixin is not compatible with ours, Terrestria's Andisol Farmland may fail to work correctly when both mods are loaded.
